### ISSUE TITLE
Multiple stats doesn't work

### ISSUE THREAD

**Issue by lizryzhko** [statoscope/statoscope#91]
**Describe the bug**
If pass additional or multiple stats within StatoscopeWebpackPlugin or cli version, only one of the stats is available in the UI.

**To Reproduce**
1. run ```npx statoscope generate --input path/to/stats1.json path/to/stats2.json --output path/to/report.html```
2. try to choose stats in the report or show diff
3. see that only one of the stats is available

**Expected behavior**
Both stats1.json and stats2.json are available in the report.

**Versions**:
 - Node (for non-browser bug): v14.16.0
 - Browser (for browser-bug): Chrome 91.0.4472.114 
 - @statoscope/cli: 5.6.2
 - @statoscope/webpack-plugin: 5.6.2

**Comment by smelukov** [comment]
Thanks for the issue.
Are the input files have the same name (but in different directories)?
For example:

`statoscope generate --input dir1/stats.json dir2/stats.json --output path/to/report.html`

**Comment by smelukov** [comment]
I've got this behavior only when the input files have the same filename 🤔

**Comment by lizryzhko** [comment]
Nope, files have different names

**Comment by smelukov** [comment]
Can you click the "Make report" button (on the top-right panel) please?
How many files do you see there?
It should be something like this
![image](https://user-images.githubusercontent.com/6654581/126762559-27651c06-d5e0-440a-b0d9-e62720fd413e.png)

**Comment by lizryzhko** [comment]
There are 2 files as expected

**Comment by smelukov** [comment]
> There are 2 files as expected

So you should see the "Choose stats" button
![image](https://user-images.githubusercontent.com/6654581/126769251-b0fb4d02-a4f1-4c5c-ba1c-f2c661df5fe0.png)
and the two stats on the "Diff" page
![image](https://user-images.githubusercontent.com/6654581/126769327-de122e53-aa8b-47ba-9946-dfe0237492cb.png)

**Comment by lizryzhko** [comment]
Nope, while there are two files on the "Make report" page, I can see only one when trying to choose stats:

![image](https://user-images.githubusercontent.com/28068023/126770506-db903129-bbda-4d09-b0a7-5eeda404eebe.png)
---
![image](https://user-images.githubusercontent.com/28068023/126770667-20cb8e88-03ce-42ea-a7f4-69d45e337fde.png)
---
![image](https://user-images.githubusercontent.com/28068023/126770709-a8410b35-7db8-40cc-aba4-4fa6e85dbaf5.png)
---
![image](https://user-images.githubusercontent.com/28068023/126770745-10d20a77-fa45-4c15-b5da-e8f2a230019b.png)

**Comment by smelukov** [comment]
Wow :)
That's interesting

**Comment by smelukov** [comment]
I think that I need your help to deal with it :)
Can you please insert this query into the `Make Report` page and send me the result?
`compilations.({hash})`

**Comment by lizryzhko** [comment]
hmm...
![image](https://user-images.githubusercontent.com/28068023/126776921-358d3b9f-2c78-4ac5-883d-97151b0809fd.png)
After I added some changes to my project and ran statoscope, I could see two stats in the report 👍

**Comment by smelukov** [comment]
I think I've got it - the problem appears when you have two same stats, with the same compilation hash
Compilation hash changes only when a bundle changes.
So I think that there must be two stats in the list with some warn message.
I'll fix it

Again, thanks for the issue )
